CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Contribute
+
+If you want to contribute something to this repo please read our contribution policy at
+
+https://www.bareos.org/en/howto-contribute.html
```

Comment:
that does not (yet) link to something reasonable.
Maybe we want to migrate the contributor section from the docs into MD format and place it here. 
We could still include it in the docs with the same mechanism that use for the CHANGELOG.md

Reply:
Ok then we need input from owner how they want to handle this.

Reply:
Decision was made during DevMeeting to keep it like it is actually and web page will soon updated

Reply:
Still you should update the link to https://www.bareos.com/community/ ,which is the redirection target of the above link.

Reply:
> Still you should update the link to https://www.bareos.com/community/ ,which is the redirection target of the above link.

Thanks I missed that redirection :-)